src/userApp/docs.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * @swagger
+ * /api/v1/users/reset/email:
+ *  post:
+ *      summary: Send the password reset link to your email
+ *      description: Reset password link must be sent to your email when valid
+ *      tags:
+ *          - Users
+ *      requestBody:
+ *          required: true
+ *          content:
+ *              application/json:
+ *                  schema:
+ *                      properties:
+ *
+ *                          email:
+ *                              required: true
+ *                              type: string
+ *                              example: test@me.com
+ *
+ *      responses:
+ *          200:
+ *              description: Link sent to your email
+ *          400:
+ *              description: Invalid email
+ *          500:
+ *              description: Internal server error
+ */
+
+/**
+ * @swagger
+ * /api/v1/users/reset/change/:token:
```

Comment:
Change this route too. And the status codes as well to include the 400 that is sent when a token has expired.
Route should be `/accounts/reset-password/:token